tools/package.json

```diff
@@ -5,18 +5,13 @@
   "author": "Dan Schulte",
   "license": "MIT",
   "scripts": {
-    "clean": "rmdir /S /Q out",
-    "test": "mocha runSchemaValidation.js"
-  },
-  "dependencies": {
-    "chalk": "^1.1.3",
-    "sync-request": "^2.1.0",
-    "tv4": "1.2.7",
-    "vscode-json-languageservice": "3.0.2",
-    "mocha-each": "^1.1.0"
+    "test": "mocha runSchemaValidation.js",
+    "test-ci": "mocha runSchemaValidation.js --reporter mocha-junit-reporter"
```

Comment:
>--reporter mocha-junit-reporter [](start = 45, length = 31)

how does this work?

Reply:
It outputs a file in a standard format (Junit test results) that Azure DevOps knows how to consume